src/views/RealTimeEvents/Drawer/tableEvents.vue

```diff
@@ -27,15 +27,45 @@
         field="value"
         header="Value"
         style="width: 60%"
-      ></Column>
+      >
+        <template #body="slotProps">
+          <div
+            v-if="slotProps.data.value?.type === 'clipboard'"
+            class="flex gap-4 items-center w-full"
+          >
+            <div
+              class="w-full max-w-[200px] sm:max-w-sm overflow-y-scroll p-2"
+              data-testid="data-table-value"
+            >
+              {{ slotProps.data.value.value }}
```

Comment:
Caso possível melhore este ponto para evitar essa redundância na chamada `.value.value`.
Talvez algo assim:
```suggestion
              <template #body="{  data }">
```

Reply:
Isso não vai evitar o value.value